dev/conformance/conformance-tests/query-invalid-operator.json

```diff
@@ -2,7 +2,7 @@
   "tests": [
     {
       "description": "query: invalid operator in Where clause",
-      "comment": "The ! operator is not supported.",
```

Comment:
Is `!` now a supported operator?

Reply:
:shrug: I just copied this from the conformance tests master repository. The test passes lol.